community/vimpc/depends

```diff
@@ -0,0 +1,4 @@
+automake make
+libmpdclient
+ncurses make
```

Comment:
Runtime dep?

Reply:
Yeah libmpdclient is a runtime dependency for all mpd clients.

Reply:
I meant `ncurses` being `make`.